frontend/components/elements/StoryItem.vue

```diff
@@ -13,45 +13,45 @@
 
         <template v-else>
             <!-- Drag and drop -->
-            <td v-if="isView(['backlog', 'planning-sprint', 'planning-backlog'])">
+            <td v-if="isView(['backlog', 'planning-sprint', 'planning-backlog'])" :class="$style.parent">
                 <nobr>
                     <BButton
-                            v-if="isView('planning-backlog')"
-                            size="sm"
-                            variant="primary"
-                            title="Add to sprint"
-                            @click="addToSprint">
+                        v-if="isView('planning-backlog')"
```

Comment:
should have another color than the 'remove from sprint' button to get the difference between them.

Reply:
I don't think that makes sense - they belong to the same category of button (determining whether the story is in the sprint or not), and they're already differentiated by icon and text - changing the colour as well would just disassociate their function from each other, which is not what we want.

Reply:
ok